### ISSUE TITLE
Jetstream broken in python 3.10

### ISSUE THREAD

**Issue by alex-janss** [tgen/jetstream#144]
Traceback (most recent call last):
  File "/efs/home/ajanss/ng2-ldt/modules/rna/env/lib/python3.10/site-packages/jetstream/runner.py", line 270, in handler
    res = future.result()
  File "/efs/home/ajanss/ng2-ldt/modules/rna/env/lib/python3.10/site-packages/jetstream/backends/slurm.py", line 228, in spawn
    job.event = asyncio.Event(loop=self.runner.loop)
  File "/efs/home/ajanss/ng2-ldt/modules/rna/env/lib/python3.10/asyncio/locks.py", line 168, in __init__
    super().__init__(loop=loop)
  File "/efs/home/ajanss/ng2-ldt/modules/rna/env/lib/python3.10/asyncio/mixins.py", line 17, in __init__
    raise TypeError(
TypeError: As of 3.10, the *loop* parameter was removed from Event() since it is no longer necessary
[2022-09-16 00:03:05]: Unhandled exception in a task future: <Task finished name='Task-5' coro=<SlurmBackend.spawn() done, defined at /efs/home/ajanss/ng2-ldt/modules/rna/env/lib/python3.10/site-packages/jetstream/backends/slurm.py:190> exception=TypeError('As of 3.10, the *loop* parameter was removed from Event() since it is no longer necessary')>

**Comment by bryce-turner** [comment]
Thanks for the report @alex-janss, this specific error should be fixed in our development branches. We will push out a release on Monday. For now, this should be resolved if you reinstall via:
```
$ pip install --user git+https://github.com/tgen/jetstream.git@feature-container-support
```
Or if cloned locally:
```
$ git fetch
$ git checkout feature-container-support
$ git pull
$ pip install --user .
```

**Comment by bryce-turner** [comment]
Fixed in release v1.7

Thanks again for the report!
